nxc/protocols/ssh.py

```diff
@@ -32,28 +44,23 @@ def proto_logger(self):
                 "hostname": self.hostname,
             }
         )
-        logging.getLogger("paramiko").setLevel(logging.WARNING)
 
     def print_host_info(self):
-        self.logger.display(self.remote_version)
+        self.logger.display(self.remote_version if self.remote_version else "Unknown SSH version, skipping...")
         return True
 
     def enum_host_info(self):
         self.remote_version = self.conn._transport.remote_version
-        self.logger.debug(f"Remote version: {self.remote_version}")
-        self.server_os = ""
-        if self.args.remote_enum:
-            stdin, stdout, stderr = self.conn.exec_command("uname -r")
-            self.server_os = stdout.read().decode("utf-8")
-            self.logger.debug(f"OS retrieved: {self.server_os}")
-        self.db.add_host(self.host, self.args.port, self.remote_version, os=self.server_os)
+        self.logger.debug(f'Remote version: {self.remote_version if self.remote_version else "Unknown SSH Version"}')
+        self.db.add_host(self.host, self.args.port, self.remote_version if self.remote_version else "Unknown SSH Version")
 
     def create_conn_obj(self):
+        logging.getLogger("paramiko").disabled = True
+        logging.getLogger("paramiko.transport").disabled = True
```

Comment:
I think these are better placed in the `proto_logger` function imo

Reply:
Fixed